include/zephyr/bluetooth/audio/audio.h

```diff
@@ -631,6 +631,24 @@ struct bt_audio_codec_cfg {
 int bt_audio_data_parse(const uint8_t ltv[], size_t size,
 			bool (*func)(struct bt_data *data, void *user_data), void *user_data);
 
+/**
+ * @brief Counts the number of set bits (1s) of an unsigned integer (32 bits)
+ *
+ * @param val The input integer for which to count set bits
+ *
+ * @return The number of set bits
+ */
+uint8_t bt_audio_count_set_bits(uint32_t val);
```

Comment:
This does not have anything to do with LE Audio. If we want to introduce such a function, it should be generic

Reply:
If you are not up for contributing such a non-audio public API function, we can just keep it static in `audio.c`, but let's not introduce it as a public `bt_audio` function :)

Reply:
Yes, that's right. It was my previous modification when i decided to use this function instead of POPCOUNT in bap_broadcast_sink_test.c file (which is no longer applicable according to the previous comments), and then I pushed the commit by mistake! So, I will update it in the next commit.